apps/api/src/app/events/e2e/echo-trigger.e2e-ee.ts

```diff
@@ -320,7 +320,7 @@ async function discoverAndSyncEcho(session: UserSession) {
   const resultDiscover = await axios.get(echoServer.serverPath + '/echo?action=discover');
 
   await session.testAgent.post(`/v1/echo/sync`).send({
```

Comment:
Let's also handle a backwards compatible rename from `/v1/echo` -> `/v1/bridge` in this PR.